users/tests.py

```diff
@@ -0,0 +1,72 @@
+# from django.test import TestCase
+# from asyncio import FastChildWatcher
+import pytest
+from django.core.exceptions import ValidationError
+from . import models
+
+
+PHONE_NUM = "0528883333"
+EMAIL = "user@gmail.com"
+PASSWORD = "password"
+NAME = "user"
+
+
+@pytest.fixture
+def user0():
+    return models.User(email=EMAIL, phone_number=PHONE_NUM, password=PASSWORD, name=NAME)
+
+
+def test_new_user(user0):
+    assert user0.email == EMAIL
+    assert user0.phone_number == PHONE_NUM
+    assert user0.name == NAME
+    assert user0.profile_pic == 'profile_pic.svg'
+    assert user0.password == PASSWORD
+
+
+def create_user(email, phone_number, password, name):
+    return models.User(email=email, phone_number=phone_number, password=password, name=name)
+
+
+@pytest.mark.parametrize("users, excpected_error", [
```

Comment:
you should consider Password Validation
(usually allowed passwords accept a password that has at least 1 capital letter and 1 symbol with minimum length of 6 digits)

Reply:
done. I have used the default password validator and added it to the pre-save function